### DISCUSSION TITLE
благодарность

### DISCUSSION THREAD

**Discussion by Tafer181** [Shabinder/SpotiFlyer#2863] [General]
I am grateful for having such a convenient application that helped me download many songs, and I can listen to them when there is no internet. However, there are some downsides as quite a few songs cannot be downloaded. Out of 150 songs, around 56 are not downloading, and some are being replaced, meaning when the song is downloaded, I receive something else.

**Comment by lyskovr** [comment]
На данный момент приложение качает музыку не с серверов Spotify, а ищет композиции с подходящими тегами в YouTube Music. Отсюда и подобные несоответствия. Нужно немного подождать. 
P.S. если нет Spotify Premium качество выше 256 ставить бесполезно

---

At the moment, the application does not download music from Spotify servers, but searches for songs with suitable tags in YouTube Music. Hence such inconsistencies. We need to wait a little.
P.S. if you doesn't have Spotify Premium, setting quality above 256 is useless
